crates/isograph_schema/src/magic_mutation_fields.rs

```diff
@@ -2,14 +2,20 @@ use common_lang_types::{
     DirectiveArgumentName, DirectiveName, IsographObjectTypeName, Location, SelectableFieldName,
     Span, StringLiteralValue, ValueKeyName, WithEmbeddedLocation, WithLocation, WithSpan,
 };
-use graphql_lang_types::{ConstantValue, GraphQLDirective, GraphQLInputValueDefinition};
+use graphql_lang_types::{
+    ConstantValue, GraphQLDirective, GraphQLInputValueDefinition, NameValuePair, ValueType,
+};
 use intern::{string_key::Intern, Lookup};
+use isograph_config::ConfigOptions;
 use isograph_lang_types::{
     DefinedTypeId, ObjectId, ResolverFieldId, ScalarFieldSelection, Selection, ServerFieldId,
     ServerFieldSelection,
 };
-
-use isograph_config::ConfigOptions;
+use serde::de::value::SeqDeserializer;
+use serde::de::MapAccess;
+use serde::de::{self, IntoDeserializer};
```

Comment:
nit: could you merge these imports? This can't be done by rustfmt on stable yet :/

Reply:
I ran cargo fmt